### PR TITLE
fix: fix gift recipient relation

### PR DESCRIPTION
**Pull request by asbiin**

### CODE REVIEW

**Comment by asbiin** [comment]
Sentry issue: [PRODUCTION-APP-74](https://sentry.io/monica-lv/production-app/issues/654011450/)

File: tests/Feature/ContactTest.php [hunk 1]
```diff
@@ -154,6 +224,51 @@ public function test_user_can_edit_a_gift_()
...
+            $gift + [
+                'is_an_idea' => true,
+                'has_been_offered' => false,
+                'contact_id' => $contact->id,
+                'account_id' => $user->account_id,
+                'is_for' => $otherContact->id,
+            ]
+        );
+
+        $new_gift = Gift::find($old_gift->id);
```
Comment (djaiss) [inline tests/Feature/ContactTest.php]: `$new_gift` → `$newGift` to be PSR-4 compliant

Comment (asbiin) [reply tests/Feature/ContactTest.php]: So, `old_gift` should be renamed with `oldGift` too?

Comment (djaiss) [reply tests/Feature/ContactTest.php]: Indeed.

**Pull request merged**